.travis.yml

```diff
@@ -4,12 +4,13 @@ python:
   - "2.7"
   - "3.3"
   - "3.4"
+  - "3.5.0"
```

Comment:
The problem is that Travis CI currently doesn't have an alias for `3.5.0` -> `3.5`. Hence you need to request `3.5.0` explicitly, for the time being. However, in order to support the longer version number I had to improve the conversion below. Which I agree, is cleaner that way, anyway.